build-scripts/validate_pr_in_jenkins

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if[ -z "$ghprbPullId" ]; then exit 0; fi
+
+allowed_section_titles='Features|Content|Interface|Mods|Balance|Bugfixes|Performance|Infrastructure|Build|I18N'
+
+echo 'Validating Summary line from pull request.'
+echo "$ghprbPullLongDescription" | grep -P "(^|\\\\r\\\\n)SUMMARY: (${allowed_section_titles}) \".*?\"(\\\\r\\\\n|$)"
+if[ $? -eq 0 ]; then
+    echo 'Summary looks good.'
+    exit 0;
+else
+    echo 'Invalid Summary line, please provide a summary similarr to the following:'
+    echo 'SUMMARY: Features "Added ability to fly."'
```

Comment:
Also, needs an `exit 1` (or any other nonzero) here to ensure the failure stops the build.